### PR TITLE
Run3-TB54 First step of transition to dd4hep for HCal TB simulation

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

First step to remove access to DDCompactView in the simulation of Hcal Testbeam setups

#### PR validation:

Tested using scripts in Geometry/HcalCommonData/test/python

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HcalTestBeamData/plugins/HcalTB02ParametersESModule.cc [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+#include "FWCore/Framework/interface/ESTransientHandle.h"
+#include "DetectorDescription/Core/interface/DDCompactView.h"
+#include "DetectorDescription/DDCMS/interface/DDCompactView.h"
+#include "Geometry/Records/interface/IdealGeometryRecord.h"
+#include "Geometry/HcalTestBeamData/interface/HcalTB02Parameters.h"
+#include "Geometry/HcalTestBeamData/interface/HcalTB02ParametersFromDD.h"
+
+#include <memory>
+
+#define EDM_ML_DEBUG
```
Comment (cvuosalo) [inline Geometry/HcalTestBeamData/plugins/HcalTB02ParametersESModule.cc:14]:
```suggestion
// #define EDM_ML_DEBUG
```

**Comment by cvuosalo** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**